src/components/project/query.helpers.ts

```diff
@@ -44,6 +45,27 @@ export function projectListFilter(query: Query, filter: ProjectFilters) {
       node('node'),
     ]);
   }
+
+  if (filter.pinned != null) {
+    if (filter.pinned) {
+      query.match([
+        node('requestingUser'),
+        relation('out', '', 'pinned', { active: true }),
+        node('node'),
+      ]);
+    } else {
+      query
+        .optionalMatch([
+          node('node'),
+          relation('in', 'rel', 'pinned', { active: true }),
+          node('requestingUser'),
+        ])
+        .with('node, requestingUser')
+        .where({
+          rel: isNull(),
+        });
```

Comment:
I believe this could simply be
```suggestion
      query.raw('where not (requestingUser)-[:pinned]->(node)`);
```